crates/frontend/src/pages/default_config/default_config.rs

```diff
@@ -459,5 +460,18 @@ pub fn modify_rows(
                 None
             }
         })
-        .collect()
+        .collect();
```

Comment:
I guess you can chain the sort with the filter_map, and then finally collect.
Not a requirement, just a suggestion.

Reply:
I believe this might affect readability a bit.